examples/kubernetes-ingress/scripts/00-create-certs.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
```

Comment:
Add logs in this file using `tests/helpers.bash:log` so users can follow what's going on while this script is being ran in case they hit an issue.

Reply:
Discussed offline